.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   plugins: ['react'],
   rules: {
     'react/react-in-jsx-scope': 'off',
+    'no-console': ['error'],
```

Comment:
can you explain the need of this line

Reply:
Instead of logging using console, we should use the logger to make logs in NDJSON format.